python/run-tests.py

```diff
@@ -54,10 +54,27 @@ def print_red(text):
 LOGGER = logging.getLogger()
 
 
-def run_individual_python_test(test_name, pyspark_python):
+def get_spark_dist_classpath():
+    original_working_dir = os.getcwd()
+    os.chdir(SPARK_HOME)
+    cp = subprocess_check_output(
+        ["./build/sbt", "-Phive", "export assembly/managedClasspath"], universal_newlines=True)
```

Comment:
@vanzin, as part of your next patch will `SPARK_DIST_CLASSPATH` just point to the `libs` directory? If so, we can remove this as part of that patch.